logger/google.go

```diff
@@ -0,0 +1,122 @@
+package logger
+
+import (
+	"encoding/json"
+
+	"cloud.google.com/go/logging"
+	"github.com/golang/protobuf/ptypes"
+	logtypepb "google.golang.org/genproto/googleapis/logging/type"
+)
+
+// Google logs to the Google cloud console logging platform.
+type Google struct {
+	// BaseEntry is used to define parameters other than Severity and Payload on the Google Logger.
+	BaseEntry logging.Entry
+	BlackList []Severity
+
+	// Logger here is what is actually doing the logging work. This interface is primarily for testing purposes.
+	// For example, we need to make sure that the logger discards all debug logs.
+	Logger interface {
+		Log(entry logging.Entry)
+	}
+}
+
+// NewGoogle returns a new Google logger using the provided client and name.
+func NewGoogle(client *logging.Client, name string) *Google {
```

Comment:
How to get rich:

Step 1: `NewGoogle()`
Step 2: Profit!

No change here, just found it amusing.

Reply:
HAHAHA

Reply:
Lol